pkg/tfgen/generate_python.go

```diff
@@ -446,30 +446,16 @@ func (g *pythonGenerator) emitResourceType(mod *module, res *resourceType) (stri
 	ins := make(map[string]bool)
 	for _, prop := range res.inprops {
 		pname := pyName(prop.name)
-		ptype := pyType(prop)
 
 		// Fill in computed defaults for arguments.
 		if defaultValue := pyDefaultValue(prop); defaultValue != "" {
 			w.Writefmtln("        %s = %s", pname, defaultValue)
 		}
 
-		// Check that required arguments are present.  Also check that types are as expected.
+		// Check that required arguments are present.
```

Comment:
Why did we remove the type assertions?

Reply:
They aren't correct in the presence of `Future[T]` and `Output[T]` being used as inputs to resources, which we 1) now explicitly allow and 2) would fail all typechecks asserting that these types are exactly equal to a type T.